File: benchmarks/python/list_pipeline.py
import sys

n = int(sys.argv[1]) if len(sys.argv) > 1 else 5_000
values = []

for i in range(n):
    values.append(i)

total = 0
for value in values:
    if value % 5 == 0:
        total += value

print(total)

File: benchmarks/php/list_pipeline.php
<?php
$n = isset($argv[1]) ? (int) $argv[1] : 5000;
$values = [];

for ($i = 0; $i < $n; $i++) {
    $values[] = $i;
}

$total = 0;
foreach ($values as $value) {
    if ($value % 5 === 0) {
        $total += $value;
    }
}

echo $total, PHP_EOL;
